src/client/method-unit-actions.c

```diff
@@ -308,6 +308,110 @@ static int method_thaw_unit_on(Client *client, char *node_name, char *unit) {
         return 0;
 }
 
+static int method_reset_failed_on_units(UNUSED Client *client, UNUSED Command *commands) {
```

Comment:
The input parameters are used, so no need for `UNUSED`. Please also remove the `UNUSED` also from other places where the parameters are used. 

```suggestion
static int method_reset_failed_on_units(Client *client, Command *commands) {
```